### PR TITLE
Feature: use `.env` to configure API Keys etc

### PR DESCRIPTION
**Pull request by JoeSchr**
building from the great work of [@lozanoSergio](https://github.com/lozanoSergio/node-binance-trader) I added the feature to read private config data from a `.env` file which gets ignored by git so people don't commit their secret. also it makes it easier to create PR and merge stuff...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,15 @@ npm i --unsafe-perm
 
 # Usage ⚡️
 
-Before everything please review the source code of the JS scripts (server.js, trader.js) and add your key information (BvA, Binance, etc...)
+Before everything please review the source code of the JS scripts (server.js, trader.js) and then add your API keys to `.env`
+
+To kickstart you, just copy the `.env.example` and edit relevant values like API Keys for BvA, Binance, etc...:
```
Comment (dargmuesli) [inline README.md:44]:
```suggestion
To kickstart, just duplicate the `.env.example`, name it  `.env` and insert your secret values:
```

File: README.md [hunk 2]
```diff
@@ -39,7 +39,15 @@ npm i --unsafe-perm
 
 # Usage ⚡️
 
-Before everything please review the source code of the JS scripts (server.js, trader.js) and add your key information (BvA, Binance, etc...)
+Before everything please review the source code of the JS scripts (server.js, trader.js) and then add your API keys to `.env`
```
Comment (dargmuesli) [inline README.md:42]:
```suggestion
Before everything, please review the source code of the JS scripts (server.js, trader.js) and then add your secret data to `.env`.
```

File: README.md [hunk 3]
```diff
@@ -39,7 +39,15 @@ npm i --unsafe-perm
...
+
+To kickstart you, just copy the `.env.example` and edit relevant values like API Keys for BvA, Binance, etc...:
+```bash
+cp .env.example .env
+$EDITOR .env
+```
+**Never check in your `.env` file!**
+It contains your most private information
```
Comment (dargmuesli) [inline README.md:50]:
```suggestion
It contains your most private information.
```

File: backtest.js [hunk 4]
```diff
@@ -18,6 +19,8 @@ const max_rows = 600000
 const showReport = true
 const trading_fees = 0.20
 
+console.log(pg_connectionString)
```
Comment (dargmuesli) [inline backtest.js:22]: This could visibly print secret data to the command line, e.g. the postgres user's password.

File: trader.js [hunk 5]
```diff
@@ -8,24 +8,15 @@ const axios = require('axios')
...
-//////////////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////////////
-
-const bva_key = "replace_with_your_BvA_key" 
-
+const env = require('./env')
 
 //////////////////////////////////////////////////////////////////////////////////
 //////////////////////////////////////////////////////////////////////////////////
 //         PLEASE EDIT PREFERENCES BELOW
```
Comment (dargmuesli) [inline trader.js:15]: There is nothing to edit anymore, is it?

**Review by dargmuesli — CHANGES_REQUESTED** [review]
Great feature! 🥰

File: trader.js (hunk 5 above)
Comment (JoeSchr) [reply trader.js:15]: d'accord

**Comment by JoeSchr** [comment]
@dargmuesli incorporated your changes. thanks for reviewing

**Comment by dargmuesli** [comment]
Are oyu

**Comment by JoeSchr** [comment]
@dargmuesli sure, you can find me in the graphile discord, as well as the other usual suspects: nuxtjs, vue land,typescript,vuetify,etc. Same username ;D.

**Pull request merged**